README.md

```diff
@@ -12,7 +12,7 @@ SURI has an [OpenAPI 3.0 spec](http://spec.openapis.org/oas/v3.0.2) that documen
 
 ## Requirements
 
-1. Ruby (3.2.0 or greater)
+1. Ruby 3.4.x
```

Comment:
Suggest removing this from the README. Unless the app deviates from our default, mentioning in the README is likely to be out of date.

Reply:
I don't find having this in the README very helpful, so I'm inclined to agree.